src/backend/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -86,6 +96,10 @@ Categories=Game;
       }
       break
     }
+    case 'darwin': {
+      await generateMacOsApp(gameInfo)
+      break
+    }
     default:
```

Comment:
This default case can now be removed, right?

Reply:
I was thinking about it. But some people are using heroic on chrome os. Do you know if they are recognized as Linux as well? I have no idea. So I left it there just in case.

Reply:
ChromeOS is identified as Linux. At least there isn't a possible value for it in [the documentation](https://nodejs.org/api/process.html#processplatform)

Reply:
ok then